lightning_uq_box/models/bnnlv/utils.py

```diff
@@ -105,7 +105,7 @@ def retrieve_module_init_args(
     Returns:
         nn.Modules init args
     """
-    current_init_arg_names = list(inspect.signature(current_module.__init__).parameters)
+    current_init_arg_names = list(inspect.signature(current_module.__init__).parameters)  # type: ignore[misc]
```

Comment:
inspecting the function signature is not recommended, but I didn't know a better solution off the top of my head, so ignored this warning for now